### PR TITLE
Move sends to tools

### PR DESCRIPTION
**Pull request by Hinton**
## 📔 Objective

The Sends feature is currently owned by the tools team, however in the SDK it's under Vault. Refactoring the code and extracting Sends as a top level client, and moving the code to the appropriate folder.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation
  team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: crates/bitwarden/src/mobile/tool/client_sends.rs [hunk 1]
```diff
@@ -87,10 +87,8 @@ impl<'a> ClientSends<'a> {
     }
 }
 
-impl<'a> ClientVault<'a> {
-    pub fn sends(&'a self) -> ClientSends<'a> {
-        ClientSends {
-            client: self.client,
-        }
+impl<'a> Client {
```
Comment (dani-garcia) [inline crates/bitwarden/src/mobile/tool/client_sends.rs:90]: Are we not making a ClientTools sub-client for them, to match `vault` and `platform`?

File: crates/bitwarden/src/mobile/mod.rs [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#[cfg(feature = "internal")]
 pub mod crypto;
 pub mod kdf;
+pub mod tool;
```
Comment (dani-garcia) [inline crates/bitwarden/src/mobile/mod.rs:3]: Any reason to call the modules `tool` instead of `tools`?

Comment (Hinton) [reply crates/bitwarden/src/mobile/mod.rs:3]: The non module moduled is called `tool`. But other than that no.

File: crates/bitwarden/src/mobile/tool/client_sends.rs (hunk 1 above)
Comment (Hinton) [reply crates/bitwarden/src/mobile/tool/client_sends.rs:90]: I'm a bit split, the tools team hace more well defined features than the other teams. And we currently expose generators and exporters directly on client.

If we decide to break out sends from tools this means we wouldn't have to refactor consumers code as much.

**Review by dani-garcia — APPROVED** [review]

**Pull request merged**